### PR TITLE
Login css

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

**Comment by BruceJi7** [comment]
This pull request is 32 files, it's shuge 😁

File: src/components/UserBox/UserBox.module.scss [hunk 1]
```diff
@@ -16,4 +14,9 @@
       font-size: 0.8rem;
     }
   }
+
+  & .ProfilePicture {
+    border: solid 3px #fff !important;
```
Comment (BruceJi7) [inline src/components/UserBox/UserBox.module.scss:19]: I assume that !important is the only way to get this to work?
If it's something we have control over, you could also try changing the order the classes are applied

File: src/hooks/useLogin.tsx [hunk 2]
```diff
@@ -18,9 +18,14 @@ function useLogin() {
     isError,
   } = useMutation(
     async (password: string) => {
+      console.log(getIdentity(), "get identity");
```
Comment (BruceJi7) [inline src/hooks/useLogin.tsx:21]: I spy a console log!

**Review by BruceJi7 — APPROVED** [review]
The only thing is there is a stray console.log :D nice

File: src/pages/Intro/PasswordSetup/index.tsx [hunk 3]
```diff
@@ -102,7 +102,7 @@ function PasswordSetup() {
           register={register("password", {
             required: { value: true, message: "Password is required" },
             minLength: {
-              value: 5,
+              value: 6,
               message: "Password must be at least 5 characters long",
```
Comment (ivanms1) [inline src/pages/Intro/PasswordSetup/index.tsx:106]: change error message too

**Review by ivanms1 — CHANGES_REQUESTED** [review]
just some small changes

File: src/components/UserBox/UserBox.module.scss (hunk 1 above)
Comment (JM066) [reply src/components/UserBox/UserBox.module.scss:19]: The css from google was unoverwritable so I used !important  😁 Initially I made an outer circle but it was not round? enough.

**Comment by ivanms1** [comment]
@JM066 remember to resolve the conversations once you solved the issues

File: package.json [hunk 4]
```diff
@@ -30,6 +30,7 @@
     "react-query": "^3.23.2",
     "react-switch": "^6.0.0",
     "react-toastify": "^7.0.4",
+    "react-tooltip": "^4.2.21",
```
Comment (BruceJi7) [inline package.json:33]: I'll approve it with this in, but we have to experiment with this package, it may work okay and maybe not. Let's see!

**Review by BruceJi7 — APPROVED** [review]

**Comment by BruceJi7** [comment]
Haha Ivan approve yo changes man @ivanms1

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**